cpp/cugraph_module/mg_cugraph_utility.hpp

```diff
@@ -75,19 +84,29 @@ auto CreateCugraphFromMemgraph(const mg_graph::GraphView<> &mg_graph, const mg_g
   rmm::device_uvector<TVertexT> cu_vertices(mg_vertices.size(), stream);
   raft::update_device(cu_vertices.data(), mg_vertices.data(), mg_vertices.size(), stream);
 
+  // TODO(gitbuda): Update types and inject valid ids / edge types.
+  rmm::device_uvector<int64_t> cu_edge_ids(mg_edge_ids.size(), stream);
+  raft::update_device(cu_edge_ids.data(), mg_edge_ids.data(), mg_edge_ids.size(), stream);
+  rmm::device_uvector<int32_t> cu_edge_types(mg_edge_types.size(), stream);
```

Comment:
edge type is int32_t if prebuilt libraries are use
but since this is not used in our current use-cases, delete this part because it will just do useless work